### PR TITLE
const2bit: avoid heap allocations

### PR DESCRIPTION
**Pull request by sago35**
This PR removes heap allocation from `const2bit`.
It is considered safe since it essentially uses the same approach as #46.

### CODE REVIEW

**Comment by sago35** [comment]
I will revise the commit message to align with the others, so I'll mark it as a draft for now. The code is already complete, so it's fine to review it if you'd like.

**Comment by sago35** [comment]
Now ready for review.

File: const2bit/const2bit.go [hunk 1]
```diff
@@ -82,6 +83,8 @@ func (f *Font) GetYAdvance() uint8 {
 }
 
 // GetGlyph returns the glyph corresponding to the specified rune in the font.
+// Since there is only one glyph used for the return value in this package,
+// concurrent access is not allowed. Normally, there is no issue when using it from tinyfont.
```
Comment (deadprogram) [inline const2bit/const2bit.go:87]: Could lock it behind a `sync.Mutex` if needed in the future.

**Comment by deadprogram** [comment]
Thanks for the improvement @sago35 now merging.

**Pull request merged**